### DISCUSSION TITLE
Why İt is not working?

### DISCUSSION THREAD

**Discussion by Rstarks07** [null2264/CobbleGen#15] [Q&A] [answered]
Why it is not working

![image](https://user-images.githubusercontent.com/107148716/229797463-b515a354-e8ac-4723-bd12-dd6661ade1e3.png)
[cobblegen.txt](https://github.com/null2264/CobbleGen/files/11148716/cobblegen.txt)

**Comment by null2264** [comment, marked as answer]
I assume that the file attached is just a portion of your config?
If not you should wrap it with another `{}`, like this:

```json5
{ // <-
	"customGen": {
		"cobbleGen": {
			"minecraft:end_stone": [
				{
					"id": "minecraft:end_stone",
					"weight": 100.0,
				}
			],
			"minecraft:granite": [
				{
					"id": "minecraft:granite",
					"weight": 100.0,
				}
			]
		}
	},
} // <-
```
